epoxy-adapter/src/main/java/com/airbnb/epoxy/DebugTimer.java

```diff
@@ -14,10 +15,11 @@
 
   private void reset() {
     startTime = -1;
+    sectionName = null;
   }
 
   @Override
-  public void start() {
+  public void start(String sectionName) {
```

Comment:
this isn't used, did you mean to assign the field?

Reply:
whoops, was copying over from other repo